app.py

```diff
@@ -154,6 +82,7 @@ def execute():
         success = True
     except redis.exceptions.ConnectionError:
```

Comment:
I hope that is the exception thrown when the password is wrong, did you verify that?